src/azure-cli/HISTORY.rst

```diff
@@ -5,6 +5,7 @@ Release History
 
 **ACR**
 
+* Disable the pull request trigger by default for ACR Tasks
```

Comment:
I suggest you call out this is a breaking change by starting with `[BREAKING]`

Reply:
See fix.